Assets/Scripts/ProjectileScript.cs

```diff
@@ -0,0 +1,63 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+// Script to control the projectiles once they are spawned.
+public class ProjectileScript : MonoBehaviour
```

Comment:
Not for this review, but probably for Freddie, but it would be a good idea to create a parent abstract class, with some common behavior methods, and then each bomb implementing specific ways of detonate, etc...

Reply:
Yea, I think inheritance/interfaces/ or composition could be ways to approach this behavior

Reply:
yeah, that is what I was thinking as well